quickjs.c

```diff
@@ -6765,7 +6765,9 @@ static void build_backtrace(JSContext *ctx, JSValue error_val, JSValue filter_fu
     saved_exception = JS_GetException(ctx);
 
     // Extract stack trace limit.
-    JS_ToFloat64(ctx, &d, ctx->error_stack_trace_limit);
+    if (JS_ToFloat64(ctx, &d, ctx->error_stack_trace_limit) < 0) {
+        // Ignore error since it sets d to NAN anyway.
+    }
```

Comment:
Maybe better:
```suggestion
    // Ignore error since it sets d to NAN anyway.
    // coverity[check_return]
    JS_ToFloat64(ctx, &d, ctx->error_stack_trace_limit);
```
Clearer and, from what I remember from node's coverity scans, "using" the value like you do here doesn't work.

Reply:
If that works, great!

Reply:
@trufae how could be test this?